### PR TITLE
created flexbox layout

### PR DESCRIPTION
**Pull request by eithri**

### CODE REVIEW

**Comment by juancho11gm** [comment]
Please keep the default images for this exercise.

File: 5-flexbox/styles.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+body {
+  background-image: url("https://i.pinimg.com/originals/3c/7f/b2/3c7fb260e97c2a965be1d44b8a865677.jpg");
+  background-size: cover;
+  font-family: "Oswald", sans-serif;
+}
+
+.common {
+  background-image: url("https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcReQI1pQchsDYaH-pjrrKz8HWrVskf1oriYVioUpus_Mpmuqbx4&usqp=CAU");
+}
```
Comment (juancho11gm) [inline 5-flexbox/styles.css:22]: is this class being used?

File: 5-flexbox/styles.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  font-size: 15px;
+  margin: 0 5%;
+  position: relative;
+  text-align: center;
+  top: 180px;
+}
+
+main section {
+  display: flex;
+  flex-direction: row;
```
Comment (juancho11gm) [inline 5-flexbox/styles.css:74]: `row ` it's the value by default.

**Comment by juancho11gm** [comment]
It looks very nice!.
image

One thing that you can improve testing with mobile: 
image

What if you try:
```
img {
  border: 5px double black;
  height: auto;
  width: 100%;
}
```

File: 5-flexbox/styles.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+.hero-button {
+  background-color: #217d51;
+  border: 2px solid #63afb8;
+  border-radius: 5px;
+  bottom: 15px;
+  color: white;
+  font-family: "Oswald", sans-serif;
+  left: 40%;
+  padding: 5px 10px;
+  position: absolute;
```
Comment (pushdev-code) [inline 5-flexbox/styles.css:45]: this button should not be absolute, you can do: `margin: 0 auto;`  to center it on the content.

File: 5-flexbox/styles.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+img {
+  border: 5px double black;
+  height: auto;
+  position: relative;
```
Comment (pushdev-code) [inline 5-flexbox/styles.css:51]:
```suggestion
```

Comment (eithri) [reply 5-flexbox/styles.css]: ok, i fixed already these parts

File: 5-flexbox/index.html [hunk 5]
```diff
@@ -28,7 +28,7 @@ <h1>The heroes:</h1>
         </p>
         <h2>Legendary Heroes</h2>
         <section>
-            <article class="legendary">
+            <article class="legendary card ">
```
Comment (juancho11gm) [inline 5-flexbox/index.html:31]:
```suggestion
            <article class="legendary card">
```

**Pull request merged**